Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionDrawerBase.cs

```diff
@@ -108,8 +108,8 @@ private static string GetPropertyTitle(SerializedProperty property)
                 propertyTitleNumeral = $" {property.GetIndexOfArrayElement()}";
 
             return property.type == nameof(InputActionMap) ?
-                $"Input Action Map{propertyTitleNumeral}" :
-                $"Input Action{propertyTitleNumeral}";
+                $"{property.displayName}{propertyTitleNumeral}" :
+                $"{property.displayName}{propertyTitleNumeral}";
```

Comment:
This doesn't quite work for serialized collections of InputActions. The names of the actions can be 
  *Element 0 0
  *Element 1 1
 etc

Reply:
Fixed.

Makes me think setting the name of an action should be part of the property popup in this case. So you can actually set names on all the actions and maps instead of just having numbered elements.

Reply:
Would make sense